src/Pages/Overview/Overview.js

```diff
@@ -219,17 +244,17 @@ export default function Overview(props) {
             <thead>
               <tr>
                 {[
-                  { title: 'Name/Email', className: 'text-base text-white/70' },
-                  { title: 'Printing', className: 'text-base text-white/70 hidden text-center md:table-cell' },
-                  { title: 'Verified', className: 'text-base text-white/70 text-center hidden sm:table-cell' },
-                  { title: 'Membership', className: 'text-base text-white/70 hidden text-center sm:table-cell' },
-                  { title: 'Delete', className: 'text-base text-white/70 text-center' },
-                ].map(({ title, className }) => (
+                  { title: 'Name/Email', className: 'text-base text-white/70', columnName: 'email'},
+                  { title: 'Printing', className: 'text-base text-white/70 hidden text-center md:table-cell', columnName: 'pagesPrinted'},
+                  { title: 'Verified', className: 'text-base text-white/70 text-center hidden sm:table-cell', columnName: 'emailVerified'},
+                  { title: 'Membership', className: 'text-base text-white/70 hidden text-center sm:table-cell', columnName: 'accessLevel'},
+                  { title: 'Delete', className: 'text-base text-white/70 text-center', columnName: 'delete'},
+                ].map(({ title, className, columnName }) => (
```

Comment:
for columns we dont want to sort by, skip adding the `columnName` field. we can just default to null here
```suggestion
                ].map(({ title, className, columnName = null }) => (
```

Reply:
resolved w/recent commit